src/services/accounts/AccountsAssetsService.spec.ts

```diff
@@ -0,0 +1,107 @@
+import { sanitizeNumbers } from '../../sanitize/sanitizeNumbers';
+import { blockHash789629, mockApi } from '../test-helpers/mock';
+import { AccountsAssetsService } from './AccountsAssetsService';
+
+const accountsAssetsService = new AccountsAssetsService(mockApi);
+
+describe('AccountsAssetsService', () => {
+	const at = {
+		hash: '0x7b713de604a99857f6c25eacc115a4f28d2611a23d9ddff99ab0e4f1c17a8578',
+		height: '789629',
+	};
+
+	describe('AccountsAssetsService.fetchAssetBalances', () => {
+		it('Should return the correct response with the assets param', async () => {
+			const expectedResponse = {
+				at,
+				assets: [
+					{
+						assetId: '10',
+						balance: '10000000',
+						isFrozen: false,
+						isSufficient: true,
+					},
+					{
+						assetId: '20',
+						balance: '20000000',
+						isFrozen: true,
+						isSufficient: true,
+					},
+				],
+			};
+
+			const response = await accountsAssetsService.fetchAssetBalances(
+				blockHash789629,
+				'0xffff',
```

Comment:
Any significance to this number, or could it be any string for the sake of the test and the mock return values?

Reply:
No significance, just a placeholder.